azure-go-aks/main.go

```diff
@@ -120,7 +99,22 @@ func main() {
 			return err
 		}
 
-		ctx.Export("kubeconfig", getKubeconfig(ctx, cluster, resourceGroup))
+		creds := containerservice.ListManagedClusterUserCredentialsOutput(ctx,
+			containerservice.ListManagedClusterUserCredentialsOutputArgs{
+				ResourceGroupName: resourceGroup.Name,
+				ResourceName:      cluster.Name,
+			})
+
+		kubeconfig := creds.Kubeconfigs().Index(pulumi.Int(0)).Value().
+			ApplyT(func(encoded string) string {
+				kubeconfig, err := base64.StdEncoding.DecodeString(encoded)
```

Comment:
Looking good I'm just paranoid about go runtime errors , sometimes Go compiles but throws a runtime error; running this would be invaluable (unless it's included in CI).